ingress/controllers/nginx/controller.go

```diff
@@ -870,13 +870,22 @@ func (lbc *loadBalancerController) createServers(data []interface{}) map[string]
 	servers := make(map[string]*nginx.Server)
 
 	pems := lbc.getPemsFromIngress(data)
-	if lbc.defSSLCertificate != "" {
-		ngxCert, err := lbc.getPemCertificate(lbc.defSSLCertificate)
-		if err == nil {
-			pems["_"] = ngxCert
-		} else {
-			glog.Warningf("%v", err)
-		}
+
+	var ngxCert nginx.SSLCert
+	var err error
+
+	if lbc.defSSLCertificate == "" {
+		// use system snake-oil certificate as default.
```

Comment:
please substitute snake-oil with "certificated generated at image build time". I think their choice of that option is obtuse enough.